test/integration/testHTAP.js

```diff
@@ -7,50 +7,63 @@ const async = require('async');
 const connOption = require('./connectionOptions').valid;
 const testUtil = require('./testUtil');
 
+function getRandomDBNames() {
+  const dbName = 'qcc_test_db';
+  const arr = [];
+  const randomNumber = Math.floor(Math.random() * 10000);
```

Comment:
instead of taking generating random number we can use uuid as a suffix - it should be random enough

Reply:
Using uuid instead.

Reply:
It seems like uuid has a problem in order to use it as a dbName. db663b9a3d-40a0-4baf-90c1-aaaaed9fb86d: {"name":"OperationFailedError","code":"001003","message":"SQL compilation error:\nsyntax error line 1 at position 43 unexpected '-'.","sqlState":"42000","data":{"internalError":false,"errorCode":"001003","age":0,"sqlState":"42000","queryId":"01af3b8e-0506-b1e4-0000-53b1070a75ce","line":-1,"pos":-1,"type":"COMPILATION"}}